### PR TITLE
[2-2/3] Remove data migration

### PR DESCRIPTION
**Pull request by despo**
## Context
Remove data migration introduced in #4658 so that column can be removed.

## Link to Trello card

https://trello.com/c/KtL53Ipv/3642-2-remove-the-subject-field-from-notes-stop-reading-from-subject-field

## Things to check

- [x] This code does not rely on migrations in the same Pull Request
- [x] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [x] API release notes have been updated if necessary
- [x] This code does not rely on the addition/removal of Azure config environment variables in the same Pull Request
- [x] New environment variables have been [added to the Azure config](https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/docs/environment-variables.md#azure-hosting-devops-pipeline)

### CODE REVIEW

**Comment by richardpattinson** [comment]
Happy to approve once migration has been run.

File: spec/services/data_migrations/prefix_note_subject_to_message_spec.rb
Comment (cwrw) [inline spec/services/data_migrations/prefix_note_subject_to_message_spec.rb:4]: ⭐

**Review by cwrw — APPROVED** [review]

**Comment by despo** [comment]
@cwrw no problem. I don't plan to merge this in until Monday as the other PR has just been merged and.I want to make sure the migration has ran first.

**Pull request merged**
